api_gateway/src/main/kotlin/org/thechance/api_gateway/endpoints/PermissionRoute.kt

```diff
@@ -14,8 +15,20 @@ import org.thechance.api_gateway.util.Role
 
 fun Route.permissionRoutes() {
 
+    val identityService: IdentityService by inject()
     val restaurantService: RestaurantService by inject()
 
+    route("/dashboard"){
+
+            authenticateWithRole(Role.DASHBOARD_ADMIN) {
+                get("/restaurant-request") {
```

Comment:
I believe this route was added by mistake and is not being used. You can find it in the develop and main branches, under the route ("/permission")

Reply:
i didn't add it was already there